static/app/components/events/contexts/contextCard.tsx

```diff
@@ -0,0 +1,132 @@
+import {Link} from 'react-router';
+import styled from '@emotion/styled';
+
+import {
+  getContextMeta,
+  getContextTitle,
+  getFormattedContextData,
+} from 'sentry/components/events/contexts/utils';
+import {AnnotatedTextErrors} from 'sentry/components/events/meta/annotatedText/annotatedTextErrors';
+import Panel from 'sentry/components/panels/panel';
+import {StructuredData} from 'sentry/components/structuredEventData';
+import {space} from 'sentry/styles/space';
+import type {Group, Project} from 'sentry/types';
+import type {Event} from 'sentry/types/event';
+import {defined, objectIsEmpty} from 'sentry/utils';
+import useOrganization from 'sentry/utils/useOrganization';
+
+interface ContextCardProps {
+  alias: string;
+  event: Event;
+  type: string;
+  group?: Group;
+  project?: Project;
+  value?: Record<string, any>;
+}
+
+function ContextCard({alias, event, type, project, value = {}}: ContextCardProps) {
+  const organization = useOrganization();
+  if (objectIsEmpty(value)) {
+    return null;
+  }
+  const meta = getContextMeta(event, type);
+
+  const contextItems = getFormattedContextData({
+    event,
+    contextValue: value,
+    contextType: type,
+    organization,
+    project,
+  });
+
+  const content = contextItems.map(
+    ({key, subject, value: contextValue, action = {}}, i) => {
+      if (key === 'type') {
+        return null;
+      }
+      const contextMeta = meta?.[key];
+      const contextErrors = contextMeta?.['']?.err ?? [];
+      const hasErrors = contextErrors.length > 0;
+
+      const dataComponent = (
+        <StructuredData
+          value={contextValue}
+          depth={0}
+          maxDefaultDepth={0}
+          meta={contextMeta}
+          config={{}}
+          withAnnotatedText
+          withOnlyFormattedText
```

Comment:
Instead of passing `withOnlyFormattedText` is it possible to just pass `withAnnotatedText={false}`?

Reply:
We still want to have annotated text, we just don't want some of the styling that comes packaged with them. This is for transforming `null` into italicized `<invalid>` or `<redacted>` text. The intention is that this new prop spits out primitive text that we can style now, rather than the shaded text with icon tooltips by default (since the design required us moving that to the end of the row)